toolchains/docker/toolchain.bzl

```diff
@@ -19,13 +19,15 @@ DockerToolchainInfo = provider(
     doc = "Docker toolchain rule parameters",
     fields = {
         "tool_path": "Path to the docker executable",
+        "client_config": "A custom directory for the docker client config.json",
```

Comment:
The directory containing the docker client config.json. If not specified, these rules use the one in ... (home directory?)

Reply:
It's DOCKER_CONFIG env variable followed by home directory as per https://github.com/google/containerregistry/blob/master/client/docker_creds_.py#L222

Updated the doc to reflect this but maybe